components/integrations/edit_command/__snapshots__/edit_command.test.tsx.snap

```diff
@@ -48,7 +48,7 @@ exports[`components/integrations/EditCommand should match snapshot 1`] = `
       "auto_complete": true,
       "auto_complete_desc": "auto_complete_desc",
       "auto_complete_hint": "auto_complete_hint",
-      "create_at": "1499722850203",
+      "create_at": 1499722850203,
```

Comment:
Assumed this timestamp was supposed to be a number as defined here https://github.com/mattermost/mattermost-redux/blob/master/src/types/integrations.ts